src/ui/server/controllers/insurance/your-buyer/trading-history/index.test.ts

```diff
@@ -161,21 +79,15 @@ describe('controllers/insurance/your-buyer/trading-history', () => {
   describe('HTML_FLAGS', () => {
     it('should have the correct flags defined', () => {
       const expected = {
-        CONDITIONAL_YES_HTML: BUYER.OUTSTANDING_PAYMENTS.CONDITIONAL_YES_HTML,
-        LEGEND_CLASS: `${LEGEND.S} ${FONT_WEIGHT.REGULAR}`,
+        HORIZONTAL_RADIOS: true,
+        NO_RADIO_AS_FIRST_OPTION: true,
```

Comment:
Can we make sure we have E2E test coverage for the correct order of the radios please, I mentioned in a previous comment, not sure if its for the same form though.

Reply:
Unresolving - is E2E test coverage in place for this?

Reply:
yes in trading-history.spec.js

Reply:
yes in trading-history.spec.js